Cargo.lock

```diff
@@ -3401,9 +3401,9 @@ dependencies = [
 
 [[package]]
 name = "proc-macro-hack"
-version = "0.5.19"
+version = "0.5.20+deprecated"
```

Comment:
Hmm....

Reply:
From [the `proc-macro-hack` package documentation](https://crates.io/crates/proc-macro-hack):

> Note: As of Rust 1.45 this crate is superseded by native support for #[proc_macro] in expression position. Only consider using this crate if you care about supporting compilers between 1.31 and 1.45.

Checking for the dependencies that use this crate:

<img width="844" alt="image" src="https://user-images.githubusercontent.com/18729796/210113988-65d12656-7f3f-4398-b7fa-f5d69677f849.png">

So...not a package that we can directly remove...gotta wait on upstream packages to stop using it...which likely won't happen until they stop supporting older Rust versions.

Reply:
> So...not a package that we can directly remove...gotta wait on upstream packages to stop using it...which likely won't happen until they stop supporting older Rust versions.

I highly doubt these projects are actively trying to support 1.45. That is an ancient version and will be almost impossible to support properly considering rust-lang's official crates only support the last 3 versions.

The PHF crate for example has a MSRV of 1.60 (https://github.com/rust-phf/rust-phf).

@kylewillmon Do you want to open an issue to get this fixed in our dependencies?

Reply:
It would be nice to fix, but I don't consider it much of a priority.